apps/frontend/TERMINAL_FONTS.md

```
@@ -0,0 +1,218 @@
+# Terminal Font System
+
+## Overview
+
+The Auto-Claude UI includes an embedded terminal font system that ensures consistent font rendering across all platforms without requiring system font installation.
+
+## Embedded Fonts
+
+The following fonts are embedded in the application:
+
+### 1. **JetBrains Mono** (Default)
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)